x/ibc-hooks/wasm_hook.go

```diff
@@ -300,18 +303,35 @@ func (h WasmHooks) OnAcknowledgementPacketOverride(im IBCMiddleware, ctx sdk.Con
 		return err
 	}
 
-	if !h.ProperlyConfigured() {
```

Comment:
we probably still want to call `!h.ProperlyConfigured()` to make sure the module isn't incorrectly wired (which is easy to do because the wasm keeper needs to be passed after initialization)